src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -540,9 +542,11 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                 ->addMetric(METRIC_BUILDS, 1) // per project
                 ->addMetric(METRIC_BUILDS_STORAGE, $build->getAttribute('size', 0))
                 ->addMetric(METRIC_BUILDS_COMPUTE, (int)$build->getAttribute('duration', 0) * 1000)
+                ->addMetric(METRIC_BUILDS_MB_SECONDS, $function->getAttribute('memory') * $build->getAttribute('duration', 0) * $function->getAttribute('cpus', 1))
```

Comment:
why is there no default value for `memory` ?

Reply:
Added a default of 512mb